tensorflow_estimator/python/estimator/head/sequential_head.py

```diff
@@ -19,10 +19,12 @@
 from __future__ import print_function
 
 import abc
-import collections
 
 import six
 import tensorflow as tf
+
+from six.moves.collections_abc import Iterable
```

Comment:
This fails internal tests with

```
    from six.moves.collections_abc import Iterable
ImportError: No module named 'six.moves.collections_abc'
```

Reply:
This is present from 1.13.0 of six and above : https://github.com/benjaminp/six/commit/0dc54ead62c7fb06edabf1ff22fb71c0c6d958e1 . I have reverted this back to try/except.

Reply:
We will update it later to the variant with no try/except when we can update our six dependency